adyenv6b2ccheckoutaddon/acceleratoraddon/web/src/com/adyen/v6/controllers/pages/account/AdyenStoredCardsController.java

```diff
@@ -107,9 +107,9 @@ public String removeStoredCard(@RequestParam(value = "paymentInfoId") final Stri
 
                     return REDIRECT_MY_ACCOUNT_STOREDCARDS;
                 } catch (IOException e) {
-                    LOG.error(e);
+                    LOG.error("IO Exception", e);
```

Comment:
is there any reason why in this file the LOGGER is called LOG? If it serves the same purpose I think it would be cleaner to call it the same way as elsewhere :)

Reply:
This is old code :) I don't think there is specific reason for this. I think it is good to rename it to follow correct convention, so I am updating it to LOGGER.